AdobeStockClient/Model/Client/Files.php

```diff
@@ -81,19 +97,27 @@ public function __construct(
      */
     public function execute(array $ids, array $columns, string $locale = null): array
     {
+        if (empty($ids)) {
+            throw new IntegrationException(__('Files ids can not be empty.'));
```

Comment:
I'd return an empty array instead of throwing an exception

Reply:
I provide these changes because we have an exception if 'files' are empty in a result of curl query.  Also, I think that the curl request to the Adobe Stock service with empty ids request parameter will return empty files results in exception.